tests.txt

```diff
@@ -289,4 +289,16 @@ IOStream_ReadTests_Valid_ReadInWhile1
 IOStream_ReadTests_Valid_ReadInWhile2
 Operators_PriorityTests_Valid_Ultimate
 CILTests_Valid_ConfusedLabels
-
+t_brackets
```

Comment:
Chyba zapomniałeś dorzucić pliki z testami

Reply:
@piotr-onyszczuk Możesz ustawić w opcjach repozytorium wymaganie co najmniej jednego code review przez mergem i ustawić branch **master** jako chroniony ;)

Reply:
Nie chciałem blokować zmian, tym bardziej że niektórzy już oddali więc trudniej o review. Nie jest to jakieś krytyczne repo :D.
Anyway, zrobione.

Reply:
oops rzeczywiście już dodałem. Fajnie jakby ktoś sprawdził czy expected results są ok

Reply:
> oops rzeczywiście już dodałem. Fajnie jakby ktoś sprawdził czy expected results są ok

ja dopiero wieczorkiem będę mógł jak coś